platform/admin/ui/src/main/webapp/js/views/Alerts.view.js

```diff
@@ -45,8 +46,16 @@ define([
             },
             serializeData: function () {
                 var json = this.model.toJSON();
+                json.details = _.map(json.details, function(val){
+                    if(val.message){
```

Comment:
We are actually checking for undefined here. If val or message is an empty string that is allowed.